lib/fizzy/capi.cpp

```diff
@@ -108,25 +145,20 @@ void fizzy_free_module(const FizzyModule* module)
     delete unwrap(module);
 }
 
+FizzyFunctionType fizzy_get_function_type(const FizzyModule* module, uint32_t func_idx)
+{
+    return wrap(unwrap(module)->get_function_type(func_idx));
```

Comment:
Note: `get_function_type` returns reference to `fizzy::FuncType` stored in module, then `wrap` returns pointers to its vector's data.